axonserver/src/main/java/io/axoniq/axonserver/grpc/DefaultClientIdRegistry.java

```diff
@@ -80,9 +80,6 @@ public String clientId(String clientStreamId) {
                                                .filter(e -> e.getValue().equals(clientId))
                                                .map(Map.Entry::getKey)
                                                .collect(Collectors.toSet());
-        if (current.isEmpty()) {
```

Comment:
deleting this causes exception in default implementation for ClientIdRegistry.streamIdFor, as this expects a  non-empty set.

Reply:
do you agree that make more sense to throw the exception in ClientIdRegistry.streamIdFor?